src/managerdbus.cpp

```diff
@@ -164,6 +164,22 @@ bool ManagerDBus::removeEvents(const Events &events)
     return true;
 }
 
+bool ManagerDBus::removeEvents(EventType type, const Filter &filter, const Sort &sort)
+{
+    mInterface.asyncCall("RemoveEventsBy", (int)type, filter.properties(), sort.properties());
+    return true;
```

Comment:
There's no reason for this to be a `bool` if it always returns `true`. Consider changing it to a `void` or adding error handling.